<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Daftar</class>
 <widget class="QDialog" name="Daftar">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="backlogin">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Kembali</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>118</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Daftar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>158</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="register_droneid">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="accessibleName">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(214, 214, 214);</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhEmailCharactersOnly</set>
           </property>
           <property name="inputMask">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Kode drone</string>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="registrasi_pilot">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(214, 214, 214);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Nama Pilot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="register_password">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(214, 214, 214);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Kata Sandi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="register_repassword">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(214, 214, 214);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Konfirmasi Kata Sandi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="database2">
           <property name="text">
            <string>[+] status</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>75</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="registrasi_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Daftar</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
